clerkinc/javascript
# chore(repo): Enter changesets Pre mode (for v5)

## Description


- Merge `Version packages` to produce next major version `@alpha-v5` release
- Rename references of `main-v4` to `release/v4` (are only used as reference for the future, those will actually be used to trigger flows when they are committed to `release/v4`)
- Merging to main will not publish any version

To verify `premajor` tags:
- `git checkout fix-version-packages`
- `npx changeset version`
- check Changelog and package.json changes



## Checklist

- [ ] (If applicable) [Documentation](https://github.com/clerkinc/clerk-docs) has been updated

## Type of change


## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [x] `build/tooling/chore`
